cliparser.py

```diff
@@ -33,10 +33,16 @@ def __init__(self, cli_support=None, board_names=None, add_help=True):
         super().__init__(description='PTS automation client', add_help=add_help)
 
         self.add_argument("-i", "--ip_addr", nargs="+",
-                          help="IP address of the PTS automation servers")
+                          help="IP address of the PTS automation servers."
+                          " If running with multiple servers(PTS dongles),"
+                          " specify the IP addresses separated by a space,"
+                          " e.g. \"-i 192.168.2.2 192.168.2.2\"")
```

Comment:
```suggestion
                          help="IP address of the PTS automation servers. "
                          "If running with multiple servers(PTS dongles), "
                          "specify the IP addresses separated by a space, "
                          "e.g. \"-i 192.168.2.2 192.168.2.2\"")
```

I think this is a matter of taste, but I personally prefer

You can also use the 

```python
help="""this is a long 
string that crosses 
multiple lines"""
```

If you want

Either way really works, and this is just a minor suggestion